react-ui/src/components/Login/SignIn.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Link from '@mui/material/Link';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { authenticationService } from '../../services/authentication.service';
+import logo from '../../egeria-logo.png';
+
+
+interface Props {
+}
+
+interface State {
+    isLoading: Boolean,
+    errors: Map<string, boolean>,
+    feedbackMessage: String,
+    currentUserValue: any
+}
+
+/**
+ *
+ * React component used for displaying login form.
+ *
+ * @since      0.1.0
+ * @access     public
+ *
+ */
+
+const theme = createTheme();
+
+function Copyright(props: any) {
+    return (
+        <Typography variant="body2" color="text.secondary" align="center" {...props}>
+            {'Powered by : '}
+            <Link color="inherit" href="https://egeria-project.org/">
+                Egeria
+            </Link>
+        </Typography>
+    );
+
+}
+class SignIn extends React.Component<Props, State> {
+
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            isLoading: false,
+            errors: new Map<string, boolean>(),
+            feedbackMessage: 'Sign in',
+            currentUserValue: authenticationService.currentUserValue
```

Comment:
how is this used? `currentUserValue: authenticationService.currentUserValue` if it is not used, just remove it, also from the `State` interface